hubspot3/tickets.py

```diff
@@ -78,3 +90,181 @@ def get_all(self, limit: int = -1, **options) -> list:
             offset = batch["offset"]
 
         return output if not limited else output[:limit]
+
+    def _get_batch(self, ids: List[int], extra_properties: Union[List[str], str] = None,
+                   with_history: bool = False) -> Dict[str, dict]:
```

Comment:
Does the inner `dict` value has a defined schema? If it does you could declare the type of the inner schema.

Reply:
It is a large schema that would be complex to describe there